generic/tests/cfg/nfs_corrupt.cfg

```diff
@@ -12,6 +12,7 @@
     remove_image_stg = yes
     drive_werror = stop
     drive_cache = none
+    host_dmesg_ignore = yes
```

Comment:
just one question 
host_dmesg_ignore = yes or verify_host_dmesg = no both fine for this case ?

Reply:
They are two different methods
verify_host_dmesg
https://github.com/avocado-framework/avocado-vt/blob/master/virttest/env_process.py#L1884
Whether to open the check

host_dmesg_ignore
https://github.com/avocado-framework/avocado-vt/blob/master/virttest/utils_misc.py#L3203
Whether the result of the check affects the results of the cases

We may hit other errors from host, so I think it might be better to keep the log records.

Reply:
@qingwangrh hello Is there any comment?

Reply:
No,LGTM

Reply:
Hi @zhenyzha 
I have one concern about whether checking the info of the `dmesg` is a checkpoint.
CC @qingwangrh

Reply:
Hello @YongxueHong 
Discussion with @qingwangrh , the dmesg is not checkpoint on nft_compat, so we decided to ignore this error.
The point is VM status after NFS disconnect

https://github.com/autotest/tp-qemu/blob/master/generic/tests/nfs_corrupt.py#L145

Reply:
> Hello @YongxueHong Discussion with @qingwangrh , the dmesg is not checkpoint on nft_compat, so we decided to ignore this error. The point is VM status after NFS disconnect
> 
> https://github.com/autotest/tp-qemu/blob/master/generic/tests/nfs_corrupt.py#L145

Hi @zhenyzha 
Okay, got it.
LGTM.